modules/custom/az_news/config/install/core.entity_view_display.node.az_news.az_feature.yml

```diff
@@ -0,0 +1,264 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.az_feature
+    - field.field.node.az_news.field_az_attachments
+    - field.field.node.az_news.field_az_body
+    - field.field.node.az_news.field_az_byline
+    - field.field.node.az_news.field_az_contacts
+    - field.field.node.az_news.field_az_expiration_date
+    - field.field.node.az_news.field_az_link
+    - field.field.node.az_news.field_az_main_content
+    - field.field.node.az_news.field_az_media_image
+    - field.field.node.az_news.field_az_news_tags
+    - field.field.node.az_news.field_az_published
+    - field.field.node.az_news.field_az_short_title
+    - field.field.node.az_news.field_az_subheading
+    - field.field.node.az_news.field_az_summary
+    - node.type.az_news
+  module:
+    - date_ap_style
+    - field_group
+    - smart_title
+    - text
+    - user
+third_party_settings:
+  field_group:
+    group_row:
```

Comment:
`group_row` is an `h3` but should be a `div` and have the `row` class.

Reply:
Ok. That's a two-column layout (first for text, second for the image) so we can use ml-2  on the first col. Or maybe offset-2?

Reply:
I'm not seeing the row div in the probo build at all. Is this related?

Reply:
I'll rebuild

Reply:
It's the missing row we identified yesterday (and I thought I added), but it's not in the latest commit. Working to fix now.

Reply:
Thanks for rebuilding @trackleft! It should pop up after that @cpoff 🤞